lib/tapjoy/ldap/cli/group.rb

```diff
@@ -0,0 +1,65 @@
+require_relative 'group/create'
+require_relative 'group/delete'
+require_relative 'group/add_user'
+require_relative 'group/remove_user'
+require_relative 'group/index'
+require_relative '../api/group'
+
+module Tapjoy
+  module LDAP
+    module CLI
+      # Entry point for all group subcommands
+      module Group
+        class << self
+
+          SUB_COMMANDS = %w(create delete add_user remove_user)
+
+          def commands
+            Trollop::options do
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Do not use `::` for method calls.](https://www.codacy.com/app/ali-tayarani/ldap_tools/file/2633306019/issues/source?bid=3316970&fileBranchId=3318857#l18)